<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProduct</class>
 <widget class="QMainWindow" name="NewProduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="goBack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="proSubmit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QComboBox" name="catBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="subCatBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="proName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="proPrice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="proStock">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="proBrand">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="proCat">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="proSubCat">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="proDesc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="RefreshProduct">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>360</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QComboBox" name="brandBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="selectMedia">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QListWidget" name="mediaList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>256</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
